requirements.txt

```diff
@@ -13,7 +13,7 @@ filebrowser_s3==1.0.0
 wagtail==2.0
 
 # Social auth is useful
-social-auth-core==1.1.0
+social-auth-core
```

Comment:
How come pipenv removed the version here, but not elsewhere?

Reply:
Probably my fault, not pipenv >.<